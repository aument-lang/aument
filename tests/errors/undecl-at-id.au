class A{}

def (self: A) a() {
    @a;
}