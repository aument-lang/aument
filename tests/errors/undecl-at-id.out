parser error(5) in -: unknown variable '@a'
4 |     @a;
        ^^
